database/getData.js

```diff
@@ -0,0 +1,16 @@
+const connection = require('./connect.js');
+
+const getReviews = (params, cb) => {
+  console.log('getData ran');
+  const q = 'SELECT * FROM reviews WHERE listing_id = ?';
+  console.log(typeof params);
+  connection.query(q, params, (err, res) => {
+    if (err) {
+      cb(err);
+    } else {
+      cb(null, res);
+    }
+  })
```

Comment:
You need to install a linter immediately so that you don't have to rely on your PR reviewers to tell you to add semicolons.